armory/__main__.py

```diff
@@ -367,6 +367,17 @@ def run(command_args, prog, description):
     log.debug(f"unifying sysconfig {config['sysconfig']} and args {args}")
     (config, args) = arguments.merge_config_and_args(config, args)
 
+    if args.num_eval_batches and args.index:
+        raise ValueError("Cannot have --num-eval-batches and --index")
+    if args.index and config["dataset"].get("index"):
+        log.info("Overriding index in config with command line argument")
+    if args.index:
+        config["dataset"]["index"] = args.index
+    if args.classes and config["dataset"].get("class_ids"):
+        log.info("Overriding class_ids in config with command line argument")
+    if args.classes:
+        config["dataset"]["class_ids"] = args.classes
+
```

Comment:
This is also a duplicate of what comes directly afterward. Please remove. (Shouldn't be any changes to __main__)